diff --git a/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift b/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift
index 00a985d..e386473 100644
--- a/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift
+++ b/node_modules/expo-eas-client/ios/EASClient/EASClientModule.swift
@@ -6,8 +6,8 @@ public class EASClientModule: Module {
   public func definition() -> ModuleDefinition {
     Name("EASClient")
 
-    Constants([
-      "clientID": EASClientID.uuid().uuidString
-    ])
+    Constants {
+      ["clientID": EASClientID.uuid().uuidString]
+    }
   }
 }
